backend/day-02/day-2-input.py

```diff
@@ -1,2 +1,13 @@
-age = input("Enter your age: ")
-print(f"You're {age} years old.")
\ No newline at end of file
+first_number, second_number, third_number, sum = 0, 0, 0, 0;
```

Comment:
Do you think writing each initializing each variable on one line each be more readable?

Reply:
@theadg yezzir